### PR TITLE
New Version for KSP 1.1

### PR DESCRIPTION
**Pull request by Sphinx3k**

### CODE REVIEW

**Comment by Sphinx3k** [comment]
I hope this was the right way. Still trying to figure out everything.

File: NetKAN/AtmosphereAutopilot.netkan [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-    "spec_version" : "v1.4",
+    "spec_version" : "v1.5.2",
```
Comment (plague006) [inline NetKAN/AtmosphereAutopilot.netkan]: `spec-version` doesn't change. `spec_version` is the version of the CKAN specification that a metadata file is using (for example if you're using the `find` function you need `"v1.4"`.

**Comment by plague006** [comment]
`ModularFuelTanks-5.7.0.netkan` has the contents of a .ckan file not a .netkan. A netkan requires a `$kref` to scrape information from a source.

In addition https://github.com/KSP-CKAN/CKAN-meta/blob/master/ModularFuelTanks/ModularFuelTanks-5.7.0.ckan is available already.

**Comment by plague006** [comment]
ffeba995

**Comment by plague006** [comment]
Closing this due to the referred PR/issue/commit.

**Pull request closed without merging**